CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Views/NotificationPopupPage.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using CheckDrive.Mobile.ViewModels;
+using Rg.Plugins.Popup.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
an empty line is left
if there are unused ones, it would be good to run them too

Reply:
I fixed it